### PR TITLE
SIP-31: Splitting AMM fees with stakers

### PR DESCRIPTION
**Pull request by john-light**
Current draft for `SIP-31: Splitting AMM fees with stakers`.

### 🗣 Comment on the current draft [here](https://github.com/DistributedCollective/SIPS/pull/31/files).
- If you comment on a line that already has a comment thread, please start a new comment thread unless your comment is relevant to the existing thread.
- You can share general comments or questions as a response below.  
- There is a thread in the Sovryn forum that you can also use to share your comments [here](https://forum.sovryn.app/t/draft-sip-31-splitting-amm-fees-with-stakers/1676).

### CODE REVIEW

**Comment by bitnym** [comment]
I'm heartened to see this proposal.

In my view, the Fee Proxy split should be increased to 1/3 of the total (increase Fee Proxy to 0.10%, reducing the LP split to 0.20%) to incentivize holding and locking up SOV tokens, especially given the additional risks of short selling now that SOV can be used as collateral (SIP-0028).

I also suggest explaining in the SIP how these fees are denominated (or linking to an explanation). Recall that one of the widely marketed benefits of holding and staking Sovryn's SOV token was that protocol rewards would be denominated in the settlement asset (non-inflationary).

So, e.g., in a $ETH -> $BTC trade, a fee would be denominated in $BTC and distributed to $SOV stakers and LPs.

https://twitter.com/bitnym/status/1402718292423364609

**Comment by john-light** [comment]
@bitnym 

> especially given the additional risks of short selling now that SOV can be used as collateral (SIP-0028).

Using SOV as collateral for loans does not enable short-selling SOV. Only leverage long.

> I also suggest explaining in the SIP how these fees are denominated (or linking to an explanation). Recall that one of the widely marketed benefits of holding and staking Sovryn's SOV token was that protocol rewards would be denominated in the settlement asset (non-inflationary).

I believe work is being done to convert all fees to RBTC. cc @ororopickpocket - am I remembering that correctly?

**Comment by john-light** [comment]
The draft has just been updated to reflect a change that came up during review. See the diff [here](https://github.com/DistributedCollective/SIPS/pull/31/commits/402c92f3f0fef7132ffd0f7ab33b802ff0cb27fc) for details.

**Comment by tonygallippi** [comment]
While I like the enthusiasm to encourage more staking and reward SOV stakers, I don't think that removing fees from LP providers is the right approach.  LPs take **significant risk of impermanent loss** when providing liquidity to the AMM pools.  Their chance of profit is only realized when fees generated exceed the impermanent loss of the positions.  I do not support this proposal.  I would instead support a revshare of bridge tolls or other non-AMM activities to SOV stakeholders.

File: SIP-31.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+SIP: 31
+Title: Splitting AMM fees with stakers
+Author: cowsant (@cwsnt), John Light (@john-light), Ororo (@ororopickpocket)  
+Status: Ready for vote
+Track: Contract
+Created: 2021-09-07
+---
+
+# SIP-31: Splitting AMM fees with stakers
```
Comment (tjcloa) [inline SIP-31.md:10]:
```suggestion
# SIP-0031: Splitting AMM fees with stakers
```

File: SIP-31.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+SIP: 31
```
Comment (tjcloa) [inline SIP-31.md:2]:
```suggestion
SIP: 0031
```

**Pull request merged**